toolkit/tools/imagecustomizer/docs/configuration.md

```diff
@@ -789,6 +790,57 @@ Supported options:
 
   For further details, see: https://en.wikipedia.org/wiki/BIOS_boot_partition
 
+## password type
+
+Specifies a password for a user.
+
+WARNING: Passwords should not be used in images used in production.
+
+This feature is intended for debugging purposes only.
+As such, this feature has been disabled in official builds of the Image Customizer tool.
+
+Instead of using passwords, you should use an authentication system that relies on
+cryptographic keys.
+For example, SSH with Microsoft Entra ID authentication.
+
+Example:
+
+```yaml
+os:
+  users:
+  - name: test
+    password:
+      type: locked
+```
+
+<div id="password-type-type"></div>
+
+### type [string]
+
+The manner in which the password is provided.
+
+Supported options:
+
+- `locked`: Password login is disabled for the user.
+
+Disabled options:
```

Comment:
Might be worth clarifying that they are all allowed in non-production scenarios.

Reply:
Fixed.